README.md

```diff
@@ -217,28 +216,14 @@ read the ['Financials' section](#developers) below.*
 
 #### Q: What can people expect to get *paid*?
 
-+ All developers will start on **£120** per day
-  (equates to £15 an hour for 8 hour days.)
++ All developers will start on **£120** per day 
+(equates to £15 an hour for 8 hour days)
 + Commission for Founders & Coders *does not* come out of developers' day rate
 + Within 4 months, developers will undergo a **peer led** day rate review
```

Comment:
calling it a "*day rate review*" has (*rightly*) raised the question: https://github.com/nelsonic/ateam/issues/4
we need to *contextualise* it better. Most companies have an "***HR*** **Process**" for periodic staff evaluation. (In *general*) "***HR***" people aren't (*usually*) involved in the *delivery* of work and so don't have much insight into the quality of a person's input/output beyond hearsay. This leads to the *wrong* people being promoted. Some companies have *fixed* this issue by having peer-based feedback systems which allow the people working on the project to 
We don't expect to have a "*formal*" (*tedious*) process for this because it does not scale ... We expect to use the *comments* in pull requests/issues as the first evidence of quality in work. We expect peer feedback to be *positive* and *encouraging* and foster a learning culture.